app/assets/v2/js/mautic.js

```diff
@@ -0,0 +1,169 @@
+// http://localhost:8000/api/v1/mautic/contacts?search=octavioamuchastegui@gmail.com
+// if (document.contxt.github_handle && document.contxt.env === 'prod') {
+//   mtcId
+// }
+document.addEventListener('mauticPageEventDelivered', function(e) {
+  if (document.contxt.github_handle && mtcId && !document.contxt.mautic_id) {
+    //   mtcId
+    saveMauticId(mtcId);
+  }
+
+});
+
+// `api/v1/mautic/contacts/${mtcId}/edit?includeCustomObjects=true`
+// let product = {
+//   'email': 'octavioamuchastegui@gmail.com',
+//   'customObjects': {
+//       'data': [
+//           {
+//               'alias': 'products',
+//               'data': [
+//                   {
+//                       'name': 'Kernel',
+//                       'attributes': {
+//                           'product': 'kernel'
+//                       }
+//                   }
+//               ]
+//           }
+//       ]
+//   }
+// }
```

Comment:
yes are examples, for me feels more easy to find it when working with the lib than going to the readme. Im trying to over-communicate the lib there and also on notion to avoid people not know how to use it. 
But happy to jump into another better solution